### PR TITLE
Don't add Accept-Range header on unsafe HTTP requests

### PR DESCRIPTION
**Pull request by jaytaph**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #12556 |
| License | MIT |
| Doc PR | N/A |

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php [hunk 1]
```diff
@@ -208,6 +208,24 @@ public function testDeleteFileAfterSend()
         $this->assertFileNotExists($path);
     }
 
+    /**
+     *
+     */
```
Comment (stof) [inline src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php]: this empty phpdoc should be removed

File: src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php [hunk 2]
```diff
@@ -208,6 +208,24 @@ public function testDeleteFileAfterSend()
...
+    public function testAcceptRangeOnUnsafeMethods()
+    {
+        $request = Request::create('/', 'POST');
+        $response = BinaryFileResponse::create(__DIR__.'/File/Fixtures/test.gif');
+        $response->prepare($request);
+        $this->assertEquals('none', $response->headers->get('Accept-Ranges'));
+
+        // Don't override when already set
```
Comment (stof) [inline src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php]: This should be a spearate test ``testAcceptRangeNotOverriden`` instead.

**Comment by jaytaph** [comment]
@stof fixed.

**Comment by stof** [comment]
:+1:

To maintainers: this should be merged in older branches as it is a bugfix

**Comment by fabpot** [comment]
Thank you @jaytaph.

**Pull request closed without merging**